crates/rolldown_binding/src/options/plugin/binding_plugin_options.rs

```diff
@@ -34,6 +34,7 @@ pub type BindingPluginOrParallelJsPluginPlaceholder =
 #[derive(Default)]
 pub struct BindingPluginOptions {
   pub name: String,
+  pub hook_usage: u32,
```

Comment:
Could this be infered by

```rust
let usage = ...l

if opts.build_start.is_some() {
   usage |= HookUsage::BuildStart
}
```

Reply:
I chose the number because it is auto-generated and not exposed to an outside user; this method should have better performance.